Tasks/QuickPerfTestV1/Invoke-QuickPerfTest.ps1

```diff
@@ -187,6 +187,13 @@ else
 	Write-Error ("Failed to connect to the endpoint '{0}' for VSO account '{1}'" -f $EndpointName, $VSOAccountUrl)
 }
 
+$featureFlags = @{
+	failDeprecatedBuildTask  = [System.Convert]::ToBoolean($env:FAIL_DEPRECATED_BUILD_TASK)
+}
+if ($featureFlags.failDeprecatedBuildTask)
+{
+	throw "The QuickPerfTest@1 (Cloud-based web performance test) task has been deprecated since June 4, 2019 and will soon be retired. Use the AzureLoadTest@1 task instead."
```

Comment:
Shouldn't this string be sourced from task.json?

Reply:
Same as above